conmon/conmon.c

```diff
@@ -115,6 +115,8 @@ static int opt_timeout = 0;
 static int64_t opt_log_size_max = -1;
 static char *opt_socket_path = DEFAULT_SOCKET_PATH;
 static bool opt_no_new_keyring = false;
+static char *opt_exit_command = NULL;
```

Comment:
I'm assuming there's not a man page that needs touching up?

Reply:
> I'm assuming there's not a man page that needs touching up?

[`conmon` doesn't have a man page][1] because [it's nominally an internal tool][2].

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1302#issuecomment-363236034
[2]: https://github.com/kubernetes-incubator/cri-o/pull/1302#issuecomment-363236631